package.json

```diff
@@ -16,7 +16,8 @@
     "build": "rm -rf ./dist && ./node_modules/.bin/tsc --project tsconfig-build.json",
     "lint-ts": "./node_modules/.bin/eslint --ext .ts .",
     "test": "./node_modules/mocha/bin/mocha -r ts-node/register test/*.ts",
-    "test-ci": "./node_modules/mocha/bin/mocha -r ts-node/register test/*.ts  --reporter mocha-junit-reporter"
+    "test-ci": "./node_modules/mocha/bin/mocha -r ts-node/register test/*.ts  --reporter mocha-junit-reporter",
+    "release": "standard-version"
```

Comment:
This is the only real gotcha, because there is not way to integrate with `npm version`/`npm publish` commands.

Running `yarn release` is a replacement for the former, which will update changelog and tag the repository according to the commit messages.